comet-java-client/src/main/java/ml/comet/experiment/http/ConnectionUtils.java

```diff
@@ -0,0 +1,124 @@
+package ml.comet.experiment.http;
+
+import lombok.NonNull;
+import lombok.Value;
+import org.asynchttpclient.Request;
+import org.asynchttpclient.RequestBuilder;
+import org.asynchttpclient.Response;
+import org.asynchttpclient.request.body.generator.ByteArrayBodyGenerator;
+import org.asynchttpclient.request.body.multipart.ByteArrayPart;
+import org.asynchttpclient.request.body.multipart.FilePart;
+import org.asynchttpclient.util.HttpConstants;
+import org.slf4j.Logger;
+
+import java.io.File;
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Collection of the utilities used by <code>Connection</code>.
+ */
+public class ConnectionUtils {
+    static final String FILE = "file";
+    static final String FORM_MIME_TYPE = "multipart/form-data";
```

Comment:
you can use https://docs.oracle.com/javaee/6/api/index.html for const .

Reply:
As a result of our sync up meeting, we decided to leave for now.